alerter/alerter.java

```diff
@@ -1,10 +1,14 @@
 public class Alerter {
     static int alertFailureCount = 0;
+    static int threshHold = 200;
     static int networkAlertStub(float celcius) {

```

Comment:
stub/test code and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.